docs/developer-docs/integrations/sns/testing/testing-locally.md

```diff
@@ -29,109 +29,112 @@ These scripts have been most tested with a stack that looks like this:
         <th>Stage</th>
-        <th>Example `sns-testing` command</th>
-        <th>For other developers</th>
+        <th>Example in `sns-testing`</th>
+        <th>Notes</th>
     </tr>
     <tr>
         <td>0</td>
         <td>Developers deploy a dapp to the Internet Computer</td>
-        <td><code>./deploy_test_canister.sh</code></td>
+        <td>[./deploy_test_canister.sh](https://github.com/dfinity/sns-testing/blob/main/deploy_test_canister.sh)</td>
         <td>Custom scripts used to deploy dapps (e.g. multi-canister dapps, use nix, etc...)</td>
     </tr>
     <tr>
         <td>1</td>
         <td>Dapp developers choose the initial parameters of the SNS for a dapp</td>
-        <td rowspan="4"><code>./deploy_sns.sh</code></td>
+        <td>[example_sns_init.yaml](https://github.com/dfinity/sns-testing/blob/main/example_sns_init.yaml)</td>
     </tr>
     <tr>
         <td>2</td>
-        <td>Dapp developers submit NNS proposal so they can deploy to the SNS subnet</td>
+        <td>Dapp developers add NNS root as co-controller of dapp</td>
+        <td>[./let_nns_control_dapp.sh](https://github.com/dfinity/sns-testing/blob/main/let_nns_control_dapp.sh)</td>
+        <td><code></code></td>
     </tr>
     <tr>
         <td>3</td>
-        <td>Proposal #1 (of 3) is passed or rejected</td>
+        <td>Submit NNS proposal to create SNS</td>
+        <td rowspan="1">[./propose_sns.sh](https://github.com/dfinity/sns-testing/blob/main/propose_sns.sh)</td>
     </tr>
     <tr>
         <td>4</td>
-        <td>Dapp developers trigger the SNS canisters to be created on SNS subnet</td>
+        <td>The NNS proposal is decided</td>
+        <td rowspan="5"><code>(No commands required)</code></td>
+        <td rowspan="5"><code> </code></td>
     </tr>
     <tr>
         <td>5</td>
-        <td>Dapp developers submit an SNS proposal to handover control of their dapp to the SNS</td>
-        <td rowspan="2"><code>./register_dapp.sh</code></td>
-        <td rowspan="2"> Potentially many proposals if one's dapp has many canisters</td>
+        <td>(Automated) SNS-W deploys SNS canisters</td>
     </tr>
         <tr>
         <td>6</td>
-        <td>Proposal #2 (of 3) is passed or rejected</td>
+        <td>(Automated) SNS-W sets SNS root as sole controller of dapp</td>
     </tr>
     <tr>
         <td>7</td>
-        <td>Proposal to start the decentralization swap</td>
-        <td rowspan="2"><code>./open_sns_swap.sh</code></td>
+        <td>(Automated) SNS-W initializes SNS canisters according to settings from Step 1</td>
     </tr>
     <tr>
         <td>8</td>
-        <td>Proposal #3 (of 3) is passed or rejected</td>
+        <td>(Automated) SNS swap starts</td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td>Users participate in the swap</td>
+        <td rowspan="1">[./participate_in_sns_swap.sh](https://github.com/dfinity/sns-testing/blob/main/participate_sns_swap.sh)</td>
+        <td rowspan="1"></td>
     </tr>
```

Comment:
Unfortunately I don't think linking works in tables :/ 

<img width="1020" alt="Screenshot 2023-08-24 at 2 51 12 PM" src="https://github.com/dfinity/portal/assets/15882011/09d376fd-dbb0-4d89-ac8e-a7afe0013d84">

Reply:
Also, it looks like we only have a single entry in the `Notes` section.

Reply:
Thanks for checking!
* I removed the links
* I think the one note is not pretty, but propose that we leave it as is as we plan to anyways further improve this page after 1proposal release
* I also removed the links from the next table.